### PR TITLE
Update csv to accommodate new spec x-desc

### PR DESCRIPTION
**Pull request by Avni-Sharma**

### CODE REVIEW

File: manifests/database.package.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ channels:
   name: stable
 - currentCSV: REPLACE_NAME.vREPLACE_NIGHTLY_VERSION-REPLACE_TAG
   name: nightly
-defaultChannel: stable
\ No newline at end of file
+defaultChannel: nightly
```
Comment (akashshinde) [inline manifests/database.package.yaml:7]: @Avni-Sharma  why nightly ? we can still have the stable right ?

Comment (Avni-Sharma) [reply manifests/database.package.yaml:7]: I didnt want to disrupt the stable part, so thought of nightly . Shall I switch to stable?

Comment (Avni-Sharma) [reply manifests/database.package.yaml:7]: @pmacik suggestions?

Comment (pmacik) [reply manifests/database.package.yaml:7]: Let's keep the default channel `stable` and just update changes in the `nightly`. I understand that the name `nightly` is  confusing... it should match the SBO - that is `beta`.

File: manifests/nightly/postgresql-operator.v0.0.10.clusterserviceversion.yaml [hunk 2]
```diff
@@ -1,23 +1,23 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: REPLACE_NAME.vREPLACE_VERSION-REPLACE_TAG
```
Comment (pmacik) [inline manifests/nightly/postgresql-operator.v0.0.10.clusterserviceversion.yaml:4]: Please keep the placeholders as they are properly replaced by the release process. thx.

Comment (pmacik) [reply manifests/nightly/postgresql-operator.v0.0.10.clusterserviceversion.yaml:4]: I've reverted those placeholders in https://github.com/Avni-Sharma/postgresql-operator/pull/1

File: manifests/database.package.yaml (hunk 1 above)
Comment (pmacik) [reply manifests/database.package.yaml:7]: I've renamed the `nightly` channel to `beta` to match latest SBO in https://github.com/Avni-Sharma/postgresql-operator/pull/1, while keeping the `stable` for backwards compatibility with pre-v0.2.0 versions of SBO.

**Review by pmacik — CHANGES_REQUESTED** [review]

**Comment by pmacik** [comment]
I've squashed the commits incl. the requested changes and get it all in by https://github.com/operator-backing-service-samples/postgresql-operator/pull/12

**Pull request closed without merging**